CRM/Contact/BAO/GroupContactCache.php

```diff
@@ -300,12 +300,12 @@ public static function updateCacheTime($groupID, $processed) {
    * @todo remove last call to this function from outside the class then make function protected,
    * enforce groupID as an array & remove non group handling.
    *
-   * @param int $groupID
+   * @param int $groupIDs
    *   the groupID to delete cache entries, NULL for all groups.
```

Comment:
Can we update this line of documentation as well, since it saus groupIDs is an 'int' - we should update this if in fact it could be int/null/array